deployments/Dockerfiles/autonomy/requirements.txt

```diff
@@ -3,5 +3,7 @@ requests==2.27.1
 open-aea[all]==1.31.0
 open-aea-cli-ipfs==1.31.0
 open-aea-ledger-ethereum==1.31.0
+open-aea-ledger-ethereum-flashbots==1.31.0
+open-aea-ledger-ethereum-hwi==1.31.0
```

Comment:
Will have to keep these here until the next release for the docker images to work.